platforms/access-bridge/src/http.ts

```diff
@@ -0,0 +1,97 @@
+// Allowed backend request methods.
+export const ALLOWED_REQUEST_METHODS = ['GET', 'PUT', 'POST'] as const;
+
+// Type to describe allowed request methods.
+export type RequestMethod = (typeof ALLOWED_REQUEST_METHODS)[number];
```

Comment:
I thought I added a comment about this already, but don't see it. Isn't this very similar to JwpApiService? Can we reuse it as a shared monorepo project?